packages/components/src/components/teaser/teaser.stories.ts

```diff
@@ -382,3 +382,361 @@ export const Parts = {
+ *
+ * ```html
+ * <sd-teaser>
+ *   <sd-button href="#">Link</sd-button>
+ * </<sd-teaser>
+ * ```
+ */
+
+export const SamplesNotClickable = {
+  name: 'Samples: Not clickable teaser',
+  parameters: {
+    controls: {
+      disable: true
+    },
+    backgrounds: {
+      default: 'white'
+    }
+  },
+  render: () => {
+    return html`
+      <style>
+        #teaserWithContentPlaceholder::part(media) {
+          flex-grow: 1;
+        }
+      </style>
+      <div class="flex justify-between gap-8">
+        <sd-teaser variant="primary" breakpoint="9999" inset class="flex-1">
+          <div slot="media" class="relative">
+            <img class="aspect-video object-cover" src="./placeholders/generic.jpg" alt="A generic placeholder jpg" />
+          </div>
+          <div slot="meta" class="meta-info">
+            <span class="meta-info-item">01.12.2013</span>
+            <span class="meta-info-item">| Author name</span>
+          </div>
+          <h3 slot="headline">Not clickable teaser</h3>
+          <div class="flex flex-col gap-5">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+              dolore magna aliqua.
+            </p>
+            <div class="flex-none">
+              <sd-button href="#" target="_blank" variant="primary" inverted>A SD-button</sd-button>
+              <sd-button href="#" target="_blank" variant="primary" inverted>A SD-button</sd-button>
+            </div>
+          </div>
+        </sd-teaser>
+        <sd-teaser variant="white border-neutral-400" breakpoint="9999" inset class="flex-1">
+          <div slot="media" class="relative">
+            <img class="aspect-video object-cover" src="./placeholders/generic.jpg" alt="A generic placeholder jpg" />
+          </div>
+          <div slot="meta" class="meta-info">
+            <span class="meta-info-item">01.12.2013</span>
+            <span class="meta-info-item">| Author name</span>
+          </div>
+          <h3 slot="headline">Not clickable teaser</h3>
+          <div class="flex flex-col gap-5">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+              dolore magna aliqua.
+            </p>
+            <div class="flex-none">
+              <sd-button href="#" target="_blank" variant="primary">Link</sd-button>
+              <sd-button href="#" target="_blank" variant="primary">Link</sd-button>
+            </div>
+          </div>
+        </sd-teaser>
+      </div>
+    `;
+  }
+};
+
+/**
+ * ## Clickable teaser
+ *
+ * If the teaser itself should be clickable and there are no other links inside, then wrap the teaser with an anchor tag.
+ * The button on the bottom of the teaser is not a link, it's just to make the interaction more obvious. According to
+ * the HTML spec, it is not allowed to have an anchor tag inside another anchor.
```

Comment:
```suggestion
 * If the teaser itself should be clickable and there are no other links inside, then wrap the teaser with an anchor tag.
 * The button on the bottom of the teaser is **must not be a link**. According to the HTML spec, it is not allowed to have an anchor tag inside another anchor.
```

Reply:
@solid-design-system/product-team we actually forgot the use case of a button in combination with a clickable teaser. the teaser could be a link to more details of a workshop (link on the teaser) but we need a button to add it to the shopping cart. 

I renamed the button texts and changed the description a bit. I hope this covers all we talked about ealier today. If not, lets chat tomorrow in the daily.

Reply:
I applied your suggested changes, nevertheless lets have a quick look at it after the daily :)